### PR TITLE
Complete move from APIs to Resources

### PR DESCRIPTION
**Pull request by garemoko**
We already make the big changes from Document APIs to Resource APIs. This PR finishes the job. Includes a search through the document for all occurrences of “ API”.

I also changed some instances of "endpoint" to "resource" or "resource endpoint" and some "API consumer"s to "client"s. 

Please check carefully!

### CODE REVIEW

File: xAPI.md [hunk 1]
```diff
@@ -3560,33 +3561,33 @@ in this specification do. The id is stored in the IRL, "updated" is HTTP header
 	<tr><td>contents</td><td>Arbitrary binary data</td><td>The contents of the document</td></tr>
 </table>
 
-The three Document APIs provide [document](#miscdocument) storage for learning activity 
-providers and Agents. The details of each API are found in the following sections, and the 
-information in this section applies to all three APIs.
+The three Document Resources provide [document](#miscdocument) storage for learning activity 
+providers and Agents. The details of each Resource are found in the following sections, and the 
+information in this section applies to all three Resources.
```
Comment (garemoko) [inline xAPI.md]: Should this be lowercase "resources"? (and on the line above)

File: xAPI.md [hunk 2]
```diff
@@ -4080,7 +4080,7 @@ own internal storage, or need to persist state across devices.
 The semantics of the call are driven by the stateId parameter. If it is included, 
 the GET and DELETE methods will act upon a single defined state document 
 identified by "stateId". Otherwise, GET will return the available ids, and DELETE 
-will delete all state in the context given through the other parameters. This API has
+will delete all state in the context given through the other parameters. This Resource has
```
Comment (garemoko) [inline xAPI.md]: I *think* this should be capitalized even if line 3566 is lowercase because it's talking about a specific resource?

**Comment by andyjohnson** [comment]
Conflicts here, but +1 from me.  I'm okay with "API consumer" becoming "client" until we get the naming convention down.

**Comment by andyjohnson** [comment]
Also, I think you have the capitalization correct.   Could get changed if a full tech editor gets ahold of it, but shouldn't hold up progress as I think your intent is clear.

**Comment by andyjohnson** [comment]
Merging as enough time has passed and issues are now not impacting conformance

**Pull request merged**